### PR TITLE
build universal2 wheels for py3.8

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by rgommers** [comment]
Thanks @isuruf. 

Restarting jobs that timed out.

**Comment by rgommers** [comment]
Failures are all unrelated, each job has one static typing test failure:
```
...
/venv/lib/python3.8/site-packages/numpy/typing/_generic_alias.py:46: in _parse_parameters
    for i in args:
E   TypeError: 'TypeVar' object is not iterable
        args       = +_DType_co
```

File: config.sh [hunk 1]
```diff
@@ -19,6 +19,21 @@ function build_wheel {
     wrap_wheel_builder _build_wheel $@
 }
 
+function pyinst_fname_for_version {
```
Comment (rgommers) [inline config.sh:22]: This is a copy of https://github.com/matthew-brett/multibuild/blob/devel/osx_utils.sh#L147, with a slight tweak. Having a code comment saying this overrides the same thing in `multibuild` and explaining why it's different would be useful.

**Comment by rgommers** [comment]
Great, thanks @isuruf!

**Comment by isuruf** [comment]
Should this be ported to the 1.21.x branch?

**Comment by charris** [comment]
I was planning to, I'll just merge master and make an rc2 in about a week, no need for a backport.

**Pull request merged**